test/functional/config.js

```diff
@@ -107,9 +106,6 @@ export default async function ({ readConfigFile }) {
       settings: {
         pathname: '/app/management',
       },
-      timeline: {
-        pathname: '/app/timeline',
-      },
```

Comment:
I'm a little curious how this even worked - as far as I'm aware the actual route to the app is `/app/timelion`